### PR TITLE
tmz 2.4.0 bowling points

### PR DESCRIPTION
**Pull request by t-m-z**
;-)

### CODE REVIEW

File: lib/logic/tools/games/bowling.dart [hunk 1]
```diff
@@ -0,0 +1,96 @@
+class BowlingRound {
+  final int one;
+  final int two;
+  final int three;
+
+  BowlingRound({this.one = 0, this.two = 0, this.three = 0});
+}
+
+Map<int, Map<int, int>> BOWLING_THROWS = {
```
Comment (S-Man42) [inline lib/logic/tools/games/bowling.dart:9]: This map makes no sense to me. As I see it, you need only lists from 0 to n. and not even fixed. You take n and can generate an integer list directly out of it...

**Comment by S-Man42** [comment]
Please put the logic into a logic class. Calculation of the score board should be independent from the view.

Here you have a classical case for model view concept. You have 1 model logic, but two views on it (table and score board). Please build it as this.

Dont forget to add test cases to the logic

**Comment by t-m-z** [comment]
Logic and Widget is seperated.
Test cases will follow.

File: lib/logic/tools/games/bowling.dart (hunk 1 above)
Comment (t-m-z) [reply lib/logic/tools/games/bowling.dart:9]: As I see it, for the DropDown I need a map or at leat an idea how to build the necessary map from a list.

**Comment by t-m-z** [comment]
I still have to test  as soon as this compiler error is fixed

**Pull request merged**